build.gradle

```diff
@@ -16,7 +16,7 @@ repositories {
 
 apply plugin: 'java'
 apply plugin: 'idea'
-apply plugin: 'spring-boot'
+apply plugin: 'org.springframework.boot'
```

Comment:
Why did you change this line?

Reply:
BootRun says:
"The plugin id 'spring-boot' is deprecated. Please use 'org.springframework.boot' instead."

Reply:
thanks